pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
@DeXtroTip it's better for 184 and 185 to go inside _apply_incubators (line 99), ie. once you get a reply that you successfully use an incubator on an egg, you remove it from the inventory.